doc/uftrace-dump.md

```diff
@@ -35,7 +35,7 @@ DUMP OPTIONS
 :   Show hex dump of data as well
 
 \--sample-time=*TIME*
-:   Apply sampling time when generating output for --flame-graph.  By default it
+:   Apply sampling time when generating output for --flame-graph.  By default, this command
```

Comment:
Not sure if we have to change this way. If "it" is not suitable here then "this" might be shorter and enough.